matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/WaveFormSanitizer.kt

```diff
@@ -33,7 +33,7 @@ internal class WaveFormSanitizer @Inject constructor() {
      * The array should have no less than 30 elements and no more than 120.
      * List of integers between zero and 1024, inclusive.
      */
-    fun sanitize(waveForm: List<Int>?): List<Int>? {
+    fun sanitize(waveForm: List<Int?>?): List<Int>? {
```

Comment:
This is use when we send a voice message, so I think this is not necessary to change anything in this file

Reply:
Done.